api/v1/v1_test.go

```diff
@@ -43,7 +43,12 @@ func runEnd2EndJob(pullRefs, seedRefs []string) ([]string, error) {
 		}
 	}
 
-	pushConfig := PushConfig{Registry: registryContainer.Hostname(), PathSeparator: "/", PathTemplate: "{{ .Prefix }}{{ .Path }}"}
+	pushConfig := PushConfig{
+		Registry:      registryContainer.Hostname(),
+		PathSeparator: "/",
+		PathTemplate:  "{{ .Prefix }}{{ .Path }}",
+		TagTemplate:   "{{ .Tag }}",
```

Comment:
We need to add `TagTemplate:   "{{ .Tag }}",` in specs.

Reply:
👍